ci/requirements-3.6_NUMPY_DEV.build.sh

```diff
@@ -10,5 +10,6 @@ pip uninstall numpy -y
 # install numpy wheel from master
 PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
 pip install --pre --upgrade --timeout=60 -f $PRE_WHEELS numpy scipy
+pip install -U git+git://github.com/dateutil/dateutil.git@33614e72c4beecae4821eb4012d6cb13c1bac585
```

Comment:
actually we could *always* install master I think would be fine (for this build)